src/yaws_soap_lib.erl

```diff
@@ -398,6 +398,18 @@ addSchemas([Xsd| Tail], AccModel, Options, ImportList) ->
 %%% Get a file from an URL spec.
 %%% --------------------------------------------------------------------
 get_url_file("http://"++_ = URL) ->
+    get_remote_url_file(URL);
+get_url_file("https://"++_ = URL) ->
+    get_remote_url_file(URL);
+get_url_file("file://"++Fname) ->
+    {ok, Bin} = file:read_file(Fname),
+    {ok, binary_to_list(Bin)};
```

Comment:
Overall this looks good, but we can avoid code duplication here by changing this to

    get_url_file("file://"++Fname) ->
        get_url_file(Fname);

Reply:
Sure thing. I can make the change.  This is a line I actually *didn't* touch, just git's diff was not super intelligent about it.  Very odd.  But I'm all over it. Thanks!